resources/views/livewire/importer.blade.php

```diff
@@ -191,7 +191,7 @@ class="col-md-12 table table-striped snipe-table">
 
 
 
-                                                                @if (($typeOfImport != 'location' && $typeOfImport!= 'assetModel' && $typeOfImport!= 'component' && $typeOfImport!= 'supplier') && $typeOfImport!= 'manufacturer' && $typeOfImport!= 'category' && ($typeOfImport!=''))
+                                                                @if ($typeOfImport === 'user')
```

Comment:
Should we also allow this for asset imports? Since that dynamically can create users and you might want to welcome them as well.

Reply:
I don't think that ever worked though. I think I that part was never built, so folks thought it worked but it didn't. Further down in the AssetImporter, we look for a matching user, but we're not trying to create one for some reason. We do have that in the ItemImporter (`createOrFetchUser()`) but we don't send welcomes on those since we don't actually know if the user is activated or has an email, and we wouldn't send an email without that info.

I think this definitely needs refactoring, but I also think for now, folks can keep doing what they're doing, which is sorting by created_at in the user list and bulk sending those password resets.